packages/types/src/tx/tokens/Cosmos.ts

```diff
@@ -1,22 +1,71 @@
-export type CosmosMinDenom = "uatom";
-export type CosmosTokenType = "ATOM";
+import { RegisteredCoinType, registeredCoinTypes } from "slip44";
+import { TokenInfo } from "./types";
+
+export type CosmosMinDenom = "uatom" | "uosmo";
+
+// Tokens in Cosmos ecosystem
+export const CosmosSymbols = ["ATOM", "OSMO"] as const;
+
+export type CosmosTokenType = (typeof CosmosSymbols)[number];
+type CosmosTokens = Record<CosmosTokenType, TokenInfo>;
+
 // TODO: As Cosmos tokens are added to our TokenType, map corresponding denom from Keplr config
 // See: https://github.com/chainapsis/keplr-wallet/blob/master/packages/extension/src/config.ts for all values in Keplr
-
 type CosmosDenom = [CosmosMinDenom, CosmosTokenType];
 const CosmosTokenDenoms: CosmosDenom[] = [
-  ["uatom", "ATOM"]
-]
+  ["uatom", "ATOM"],
+  ["uosmo", "OSMO"],
+];
 
-const tokenDenomLookup = (param: CosmosMinDenom | CosmosTokenType): CosmosDenom | undefined =>
-  CosmosTokenDenoms.find((tokenDenom: CosmosDenom) => tokenDenom.includes(param));
+const tokenDenomLookup = (
+  param: CosmosMinDenom | CosmosTokenType
+): CosmosDenom | undefined =>
+  CosmosTokenDenoms.find((tokenDenom: CosmosDenom) =>
+    tokenDenom.includes(param)
+  );
 
 export const tokenByMinDenom = (minDenom: CosmosMinDenom): string => {
-  const tokenDenom = tokenDenomLookup(minDenom)
-  return tokenDenom ? tokenDenom[1] : ""
-}
+  const tokenDenom = tokenDenomLookup(minDenom);
+  return tokenDenom ? tokenDenom[1] : "";
+};
 
 export const minDenomByToken = (token: CosmosTokenType): string => {
-  const tokenDenom = tokenDenomLookup(token)
+  const tokenDenom = tokenDenomLookup(token);
   return tokenDenom ? tokenDenom[0] : "";
-}
+};
+
+const supportedCoinTypes: RegisteredCoinType[] = [
```

Comment:
I think `CosmosSymbols.includes` could be used instead of `indexOf`, and the array spread can also be removed.

Reply:
Fixed!